api/generated/v2/routes.go

```diff
@@ -857,6 +936,7 @@ func (w *ServerInterfaceWrapper) SearchForTransactions(ctx echo.Context) error {
 		"address-role":          true,
 		"exclude-close-to":      true,
 		"rekey-to":              true,
+		"application-id":        true,
```

Comment:
same here - let's find one short convention.

Reply:
Looks like "index" breaking conventions. It's only used by the `Asset` model. Everything else uses "asset-id"

Reply:
Correction, there is also `created-asset-index`.